build-env/docker/docker-debian-11-bullseye/sources.list.bullseye

```diff
@@ -0,0 +1,7 @@
+deb http://deb.debian.org/debian bullseye main
+deb http://security.debian.org/debian-security bullseye-security main
+deb http://deb.debian.org/debian bullseye-updates main
+
+deb [arch=amd64] http://deb.debian.org/debian bullseye-backports main
```

Comment:
I don't think we're installing anything from backports are we? In fact I don't think we've used backports since debian stretch(9):

```bash
ryanow01@U203622:~/sandboxes/distro-pelion-edge [debian-11]$ git ls-files | xargs grep -l backport | grep -v sources.list
build-env/docker/docker-debian-9-stretch/preferences
build-env/docker/docker-ubuntu-18-bionic/00_debhelper
```

fixup PR against debian-11 branch: https://github.com/PelionIoT/distro-pelion-edge/pull/215